Building under VS2019

When I build the nunit solution under VS2019, it all succeeds except for:

```
8>------ Build started: Project: nunitlite.tests, Configuration: Debug Any CPU ------
8>C:\Projects\GitHub\nunit\src\NUnitFramework\nunitlite.tests\nunitlite.tests.csproj : warning NU3005: Package 'Microsoft.NETCore.Platforms 2.0.0' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry is invalid. The central directory header field 'compression method' has an invalid value (8).
8>C:\Projects\GitHub\nunit\src\NUnitFramework\nunitlite.tests\nunitlite.tests.csproj : warning NU3005: Package 'Microsoft.NETCore.Platforms 2.0.0' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry is invalid. The central directory header field 'compression method' has an invalid value (8).
8>C:\Projects\GitHub\nunit\src\NUnitFramework\nunitlite.tests\nunitlite.tests.csproj : warning NU3005: Package 'Microsoft.NETCore.Platforms 2.0.0' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry is invalid. The central directory header field 'compression method' has an invalid value (8).
8>C:\Projects\GitHub\nunit\src\NUnitFramework\nunitlite.tests\nunitlite.tests.csproj : warning NU3005: Package 'Microsoft.NETCore.Platforms 2.0.0' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry is invalid. The central directory header field 'compression method' has an invalid value (8).
8>C:\Projects\GitHub\nunit\src\NUnitFramework\nunitlite.tests\nunitlite.tests.csproj : warning NU3005: Package 'Microsoft.NETCore.Platforms 2.0.0' from source 'C:\Program Files\dotnet\sdk\NuGetFallbackFolder': The package signature file entry is invalid. The central directory header field 'compression method' has an invalid value (8).
```

Not sure why these errors are occurring, or what they mean?

--- a/src/NUnitFramework/Directory.Build.props
+++ b/src/NUnitFramework/Directory.Build.props
@@ -5,6 +5,7 @@
     <Features>strict</Features>
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+    <DisableImplicitNuGetFallbackFolder>true</DisableImplicitNuGetFallbackFolder>
 
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>

